### ISSUE TITLE
ui tweak suggestion - make the positional arguments as wide as possible

### ISSUE THREAD

**Issue by hanleybrand** [paololazzari/play#2]
This is a really cool tool, but I found using it for more than a very simple command was frustrating because of the positional arguments field/input area being so narrow. 

Here's an example -- Trying to get to this, which is relatively short for a jq/yq query: 

```
yq e '[.images.gitSync.repository, .images.gitSync.tag] | join(":")'
```
after I've entered the query in positional arguments, this is what it looks like

```
yq eval              'Sync.tag] | join(":") ' -- values_11.8.yml 
```

I don't know the framework at all, and have little experience with terminal apps, but if it is possible to make the positional argument input grow to available size, it would be much easier to use.  

As a secondary consideration if dynamic sizing of objects is complicated/tedious/not worth the time, maybe split the input area to 3 lines, eg something like

```
[command]  [command options (to terminal width)] 
 '<positional arguments> (to terminal width)'
 <input files> (to terminal width)
```

I did discover the wide editor and that works for sure, but it would be nice to be able to see more of the positional arguments in the default view.

This tool is cool, thanks for making it!

**Comment by paololazzari** [comment]
Hi @hanleybrand ,

Thank you for trying this out and for your feedback.

I agree with you, I think having the positional argument wider is a good idea. However any additional space given to the positional arguments will reduce that available to the file list (which I think is acceptable).

What do you think about having the positional argument automatically resize based on the length of the input? 

So by default:

![image](https://github.com/paololazzari/play/assets/6552810/edee28d0-1396-45b9-b812-1faee40069af)

then as the input grows:

![image](https://github.com/paololazzari/play/assets/6552810/229d08e8-af24-4fdf-8b88-ff0f91994635)

and finally:

![image](https://github.com/paololazzari/play/assets/6552810/4c1aafcb-75b2-4e10-b8db-d2a5bec8f166)

with this last screenshot being the maximum width possible. If the input is shortened, the same logic would apply in reverse.

**Comment by hanleybrand** [comment]
I think that sounds great - my 3 line idea was me thinking around the potentially tedious time spent getting the field to grow, but I agree the field sizing dynamically is better.

Thanks for considering my input!

**Comment by paololazzari** [comment]
This has now been implemented with version `0.3.3`.
